tests/Fixtures/TestBundle/Entity/CustomActionDummy.php

```diff
@@ -19,11 +19,11 @@
 /**
  * @ORM\Entity
  * @ApiResource(itemOperations={
- *     "get"={"method"="GET"},
- *     "custom_normalization"={"route_name"="custom_normalization"}
- * }, collectionOperations={
- *     "get"={"method"="GET"},
- *     "custom_denormalization"={"route_name"="custom_denormalization"}
+ *     "get": {"method": "GET"},
+ *     "custom_normalization": {"route_name": "custom_normalization"}
+ *     }, collectionOperations={
```

Comment:
This line shouldn't be changed. This looks like a bug, I'll open an issue on PHP CS Fixer repository.

Reply:
Indeed, fixed.